src/main/java/com/gmail/woosay333/onlinebookstore/exception/handler/ControllerExceptionHandler.java

```diff
@@ -64,6 +69,16 @@ protected ResponseEntity<Object> handleBookIsbnAlreadyExistsException(Exception
         return getResponseEntity(CONFLICT, ex.getMessage());
     }
 
+    @ExceptionHandler(AccessDeniedException.class)
+    protected ResponseEntity<Object> handleAccessDenied(AccessDeniedException ex) {
+        return getResponseEntity(FORBIDDEN, ex.getMessage());
+    }
+
+    @ExceptionHandler({JwtException.class, AuthenticationException.class})
+    protected ResponseEntity<Object> handleAuthenticationException(Exception ex) {
+        return getResponseEntity(UNAUTHORIZED, ex.getMessage());
+    }
```

Comment:
I thing this error messages handled by default

Reply:
I'm not sure. If I don't write these exceptions in GlobalExceptionHandler, I just get 401 status without any error body when authentication data is incorrect. Same thing with access rights error.